stream-access-control/access-control-core/src/main/java/com/backbase/stream/service/AccessGroupService.java

```diff
@@ -856,7 +829,7 @@ public Flux<DataGroupItem> getExistingDataGroups(String serviceAgreementInternal
     public Flux<String> getDataGroupItemIdsByServiceAgreementId(String serviceAgreementId) {
         PresentationServiceAgreementIdentifier serviceAgreementIdentifier =
             new PresentationServiceAgreementIdentifier().idIdentifier(serviceAgreementId);
-        return getDataGroupItemIds(null, serviceAgreementIdentifier);
+        return getDataGroupItemIds("ARRANGEMENTS", serviceAgreementIdentifier);
```

Comment:
@tarun-bb do you know if this would impact somehow the custom data groups for digital sales?

Reply:
@uesleilima No I don't think so since DS is also defining type of data items as Arrangements.